test/org/sagebionetworks/bridge/services/SharedModuleServiceTest.java

```diff
@@ -90,6 +90,7 @@ public void byIdAndVersionSchemaSuccess() {
 
         // mock schema service
         UploadSchema sharedSchema = UploadSchema.create();
+        sharedSchema.setPublished(true);
```

Comment:
This is not the way to test this. We need to test that we set the schema to published before we create it in the local study. What you're doing here is setting the shared schema to published. This actually makes it impossible to test that we're setting the published flag, because we can't tell whether we set the schema to published before creating it, or if it was already published to begin with.

Better to set the published flag to false, and then when verifying the call to createUploadSchema(), we verify that the passed in schema is published.